app.py

```diff
@@ -1,120 +0,0 @@
-
-import os
-import praw
-
-from flask import Flask, redirect, request, Response
-
-
-app = Flask(__name__)
-Reddit = praw.Reddit(user_agent='slack-reddit-v1 by /u/arjunblj')
-
-SORT_TYPES = {
-    'hot': 'get_hot',
-    'new': 'get_new',
-    'top': 'get_top',
-    'rising': 'get_rising'
-}
-
-
-def _create_response(text):
-    """Returns a utf-8 encoded text response.
-    """
-    return Response(text, content_type='text/plain; charset=utf-8')
-
-
-def _parse_options(options):
-    """Returns a Response after validating the input.
-    """
-    if options[1] in SORT_TYPES.keys():
-        sort_type = options[1]
-        if len(options) > 2:
-            subreddit, sort_type, num_results = options
-            try:
-                num_results = int(num_results)
-                return fetch_posts(subreddit, sort_type, num_results)
-            except ValueError:
-                # Third option is invalid.
-                response_text = ("Sorry! `%s` isn't a valid sorting number. Try `/reddit %s %s 5`"
-                                 % (subreddit, sort_type, num_results))
-                return _create_response(response_text)
-        else:
-            # No option for results to return.
-            return fetch_posts(options[0], sort_type)
-    else:
-        if isinstance(options[1], int):
-            # For the case that someone passes `/reddit [subreddit] [sort_num]`
-            return fetch_posts(options[0], 'hot', options[1])
-        else:
-            return _create_response('Invalid search. Try `/reddit aww top 5` or `/reddit help`.')
-
-
-def _get_post_header(subreddit, sort, top_num):
-    """Formats the top heading to make it a bit more readable.
-    """
-    url = 'http://reddit.com/r/' + subreddit
-    if sort == 'new':
-        heading = '%s Newest Posts from' % (top_num)
-    elif sort == 'top':
-        heading = 'Top %s Posts from' % (top_num)
-    elif sort == 'rising':
-        heading = '%s Fastest Rising Posts' % (top_num)
-    else:
-        heading = '%s Hottest Posts' % (top_num)
-    heading += ' <%s|/r/%s>\n' % (url, subreddit)
```

Comment:
Maybe some of this string formatting could have been set as global variables? Easier to find and change.